libr/include/r_core.h

```diff
@@ -927,39 +927,33 @@ extern RCorePlugin r_core_plugin_a2f;
 
 /* DECOMPILER PRINTING FUNCTIONS */
 /**
-* r_core_annotated_code_print_json() - Prints the data contained in RAnnotatedCode *code in JSON format.
-* @code: Pointer to a RAnnotatedCode
-*
-* Prints the data contained in RAnnotatedCode represented by the pointer 'code' in JSON format. 
-* The function will print the output in console using the function r_cons_printf();
-* 	
-* Return: Nothing
-*/
+ * Prints the data contained in RAnnotatedCode represented by the pointer 'code' in JSON format. 
+ * The function will print the output in console using the function r_cons_printf();
+ * @brief Prints the data contained in RAnnotatedCode *code in JSON format.
```

Comment:
I have removed the parts that say the same thing twice. I think it will be better if we keep the detailed description first and then have brief description and information regarding `parameters` and `return`.  This way, we can see the brief and parameters together. I think a detailed description won't be required every time to understand what the function does, so this is probably more readable. Also, this style is followed by many functions in other places.

Reply:
There are two arguments for having brief before more detailed description:
* That's the order Doxygen and other tools will display it. `By default the brief descriptions become the first sentence of the detailed descriptions`.  Moving it to the end means that it becomes less readable as single text.
* Typical way of marking end of brief and start of detailed description is using an empty line. Some tools like like VSCode get confused if it isn't in this order and ignores everything before brief.

> Also, this style is followed by many functions in other places.

By searching for `[@\\]brief` I was able to find only one such function in r2 which wasn't just written by you - `parse_attr_value`. And even in that case it was less of detailed description more of TODO comment noting that function is unfinished.

Reply:
@karliss I saw such examples somewhere and was also sure that I have seen it in radare2. My mistake I said many! Now I understand the problems with this and I will change it as you suggested.

Reply:
Done.